src/canvas/static_canvas.class.ts

```diff
@@ -1602,21 +1594,13 @@ export class StaticCanvas<
       vp = this.viewportTransform,
       translateX = (vp[4] - (left || 0)) * multiplier,
       translateY = (vp[5] - (top || 0)) * multiplier,
-      // @ts-ignore
-      originalInteractive = this.interactive,
       newVp = [newZoom, 0, 0, newZoom, translateX, translateY] as TMat2D,
       originalRetina = this.enableRetinaScaling,
       canvasEl = createCanvasElement(),
-      // @ts-ignore
-      originalContextTop = this.contextTop,
-      objectsToRender = filter ? this._objects.filter(filter) : this._objects;
+      objectsToRender = filter ? this._objects.filter((obj) => filter(obj)) : this._objects;
     canvasEl.width = scaledWidth;
     canvasEl.height = scaledHeight;
-    // @ts-ignore
-    this.contextTop = null;
```

Comment:
nulling the contextTop doesn't seem to have anymore any effect.

Reply:
legit

Reply:
I don't really understand when or why it did have an effect